app/models/payload.rb

```diff
@@ -29,8 +29,12 @@ def self.min_response
     minimum("responded_in")
   end
 
+  def parse_payload(params)
+    JSON.parse(params[:payload])
+  end
```

Comment:
@bfpepper Could you explain why this is a separate method? Seems like we already had it below? Was there a reason that you wanted to separate it out?